extensions/ql-vscode/src/view/common/Alert.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+type ContainerProps = {
+  type: 'warning' | 'error';
+  inverse?: boolean;
+};
+
+const getBackgroundColor = ({ type, inverse }: ContainerProps): string => {
+  if (!inverse) {
+    return 'var(--vscode-notifications-background)';
+  }
+
+  switch (type) {
+    case 'warning':
+      return 'var(--vscode-editorWarning-foreground)';
+    case 'error':
+      return 'var(--vscode-debugExceptionWidget-border)';
```

Comment:
```suggestion
      return 'var(--vscode-editorError-foreground)';
```
Changes from #a31515 to #f14c4c

Reply:
I did think about using the normal `editorError` color, but the contrast ratio between the foreground and background color is quite small with `editorError-foreground`. This makes the text somewhat hard to read. These are the results from the [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/):

![Screenshot 2022-09-26 at 13 51 14](https://user-images.githubusercontent.com/1112623/192269340-b0898840-f19f-4974-b883-973e8a2e8c48.png)

![Screenshot 2022-09-26 at 13 51 24](https://user-images.githubusercontent.com/1112623/192269381-e961484b-009c-421b-a147-4b664d2c2068.png)

Using black text instead of white text improves this somewhat, but since the text in the alerts is quite small, I still don't think it will be enough.

![Screenshot 2022-09-26 at 13 51 35](https://user-images.githubusercontent.com/1112623/192269405-47ad9b89-d8ff-4a27-bd0e-7c62bdb758f5.png)

Reply:
Thanks for checking. I didn't know about that contrast checker, but that's a very handy tool to bear in mind.